validator/src/main.rs

```diff
@@ -1134,6 +1135,7 @@ pub fn main() {
                 .long("expected-shred-version")
                 .value_name("VERSION")
                 .takes_value(true)
+                .validator(is_shred_version)
```

Comment:
I think this would work, less code:
```suggestion
                .validator(is_parsable::<u16>)
```

Reply:
this pr got literally 1-line change: https://github.com/solana-labs/solana/pull/14183/commits/6094a360bbee86cb48076ae9c8100a54391ee1a5

Reply:
perfect!